src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -337,15 +340,20 @@ public void testGetDBColumn() throws SQLException {
 
             String[] types = {"TABLE"};
             try (ResultSet rs = databaseMetaData.getTables(null, null, "%", types)) {
-
-                // Fetch one table
-                MessageFormat form1 = new MessageFormat(TestResource.getResource("R_atLeastOneFound"));
-                Object[] msgArgs1 = {"table"};
-                assertTrue(rs.next(), form1.format(msgArgs1));
-
+                if(rs.next()) {
+                    boolean tableFound = false;
+                    do {
+                        if(rs.getString("TABLE_NAME").equalsIgnoreCase(tableName)) {
+                            tableFound=true;
+                        }
+                    } while(!tableFound && rs.next());
```

Comment:
I am not sure how `tableFound` makes a difference here, it is not used to throw an exception.

Reply:
The test here iterates through all records and looks for table we just created. We don't want to test with any table but the one we just created here.

Reply:
But what if at the end of the loop, `tableFound` is `false`?

Reply:
The test would fail eventually on databaseMetadata.getColumns call.. but I updated the test to capture the variable and fail if no table found.